odoo/addons/test_translation_export/tests/test_static.py

```diff
@@ -0,0 +1,10 @@
+from odoo.tools.translate import TranslationModuleReader
+from odoo.tests import common, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestExport(common.TransactionCase):
+
+    def test_export_translatable_resources(self):
+        """Read files of installed modules and export translatable terms"""
+        TranslationModuleReader(self.env.cr, export_from_db=False, strict_mode=True)
```

Comment:
I wonder why you disable the export_from_db here 

Anyway, I'm not a big fan of adding parametter in the code to make it testable. You can monkey patch the _export_translatable_records() if you really need to disable this part.

Reply:
Same here, we are post-install so we could export from db no? Good to catch more errors.

Reply:
same here. I think we can test those resources. If so, maybe we could add a money patch for logger.exception and raise an error from the patch. So that we don't need the strict mode only for test 😃

Reply:
I don't believe we can catch errors on exporting translations from errors. It might be useful to test the export system, though. It takes just extra 15 seconds of work at my laptop. So, let's test it as well.